exercises/practice/linked-list/.meta/example.h

```diff
@@ -1,30 +1,33 @@
 #ifndef LINKED_LIST_H
 #define LINKED_LIST_H
 
-#include <stdbool.h>
+#include <stddef.h>
 
 typedef int ll_data_t;
 struct list;
 
 // constructs a new (empty) list
 struct list *list_create(void);
 
-// checks if the list is empty
-bool list_is_empty(const struct list *list);
+// counts the items on a list
+size_t list_count(const struct list *list);
 
-// inserts item at back of list
-bool list_push(struct list *list, ll_data_t item_data);
+// inserts item at back of a list
+struct list_node *list_push(struct list *list, ll_data_t item_data);
```

Comment:
Is this return value necessary? I don't think it's being used anywhere and it seems like there's very little good that a client could do with it.

Reply:
It looks like the existing implementation has been returning a `struct list_node *` but was advertised as returning a `bool` (I'm not sure how the compiler didn't get upset about this...). I think the real fix is to not return anything, though.

Reply:
@ryanplusplus I agree it seems weird to return the new node after pushing or unshifting.  I don't mind the idea of returning `bool` though.  I also don't mind the idea of some sort of enum return value that indicates the status of the operation, but I recognize that might seem like overkill (it certainly goes well beyond the spec).

Reply:
I'm fine with an addition to the spec as long as it's tested.